src/app/business/profile/createProfile/createProfile.component.html

```diff
@@ -43,18 +43,42 @@ <h1>{{I18N.keyID['sds_profile_create_title']}}</h1>
             <div class="ui-grid-col-10">
                 <div class="ui-grid ui-grid-responsive ui-grid-pad ui-fluid secondary-form">
                     <form [formGroup]="qosPolicy" [grid]="{label: 'ui-g-2', content:'ui-g-3'}" [errorMessage]="errorMessage">
-                        <form-item label="{{label.maxIOPS}}" [required]="true">
-                            <div class="input-text-vertical-align">
-                                <input pInputText type="text" formControlName="maxIOPS" />
-                                <span>{{I18N.keyID['sds_profile_IOPS_unit']}}</span>
-                            </div>
-                        </form-item>
-                        <form-item label="{{label.MBPS}}" [required]="true">
-                            <div class="input-text-vertical-align">
-                                <input pInputText type="text" formControlName="maxBWS" />
-                                <span>{{I18N.keyID['sds_profile_BWS_unit']}}</span>
-                            </div>
-                        </form-item>
+                        <div class="ui-grid-col-6">
+                            <form-item label="{{label.maxIOPS}}" [required]="true">
+                                <div class="input-text-vertical-align">
+                                    <input pInputText type="text" formControlName="maxIOPS" />
+                                    <span>{{I18N.keyID['sds_profile_IOPS_unit']}}</span>
+                                </div>
+                            </form-item>
+                            <form-item label="{{label.MBPS}}" [required]="true">
+                                <div class="input-text-vertical-align">
+                                    <input pInputText type="text" formControlName="maxBWS" />
+                                    <span>{{I18N.keyID['sds_profile_BWS_unit']}}</span>
+                                </div>
+                            </form-item>
+                            <form-item label="{{label.latency}}" [required]="true">
+                                <div class="input-text-vertical-align">
+                                    <input pInputText type="text" formControlName="latency" />
+                                    <span>{{I18N.keyID['sds_profile_latency_unit']}}</span>
+                                </div>
+                            </form-item>
+                        </div>
+                        <div class="ui-grid-col-6">
+                            <form-item label="{{label.minIOPS}}" [required]="true">
```

Comment:
Please set the new parameters (MinIOPS, MinBWS and latency) as optional. Not as required parameters.

Reply:
OK.